src/software/ai/navigator/path_manager/obstacle_path_manager.cpp

```diff
@@ -0,0 +1,79 @@
+#include "software/ai/navigator/path_manager/obstacle_path_manager.h"
+
+/**
+ * This file contains the implementation of the obstacle path manager
+ * which uses obstacles to arbitrate between paths
+ * ret a path consisting of only the start and destination
+ * points.
+ */
+
+
+ObstaclePathManager::ObstaclePathManager(std::unique_ptr<PathPlanner> path_planner)
+    : path_planner(std::move(path_planner))
+{
+}
+
+const std::map<int, Path> ObstaclePathManager::getManagedPaths(
+    const std::map<int, PathObjective> &objectives, const Rectangle &navigable_area,
+    const std::vector<Obstacle> &static_obstacles)
+{
+    std::map<int, Path> retval;
+
+    std::vector<Obstacle>
+        current_velocity_obstacles;  // velocity obstacles used to avoid collisions
```

Comment:
could we move this comment to the line above and change it to something like:
```
Velocity obstacles used to avoid collisions. 
As we plan a path for each robot, a corresponding obstacle will be added to this list so that paths planned later do not collide with the path we just planned. 
Please see: https://en.wikipedia.org/wiki/Velocity_obstacle
```

Reply:
done